odoo/http.py

```diff
@@ -19,6 +19,7 @@
 import sys
 import threading
 import time
+import urlparse
```

Comment:
It was a temp fix for an issue in master that I fixed properly at https://github.com/odoo/odoo/commit/b31435ab5df7eddd1030e59f2fb4276ed6383d98. I mistakenly left it in here.